webui/react/src/components/ResourcePoolCard.module.scss

```diff
@@ -10,6 +12,23 @@
     display: flex;
     flex-direction: column;
     flex-grow: 1;
+
+    .resoucePoolBoundContainer {
+      align-items: center;
+      border-bottom: 1px solid var(--theme-stage-border);
+      border-top: 1px solid var(--theme-stage-border);
```

Comment:
nit: I prefer using divider to have less css

Reply:
you mean antd divider? 
I tried this before the commit but it doesn't looks good since the margin of antd divider looks too wide in this case. If i don't miss anything,  there's no props to control it, so i need to add extra css to manage the margin of antd divider. I think the current implementation is cleaner.

<img width="329" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/88d42345-a86e-409f-8959-73228fa0eb33">